src/core/validators/validators.ts

```diff
@@ -145,8 +157,10 @@ const Url: ValidatorFn = (value) => {
 };
 
 const isEmptyValue = (value: Value) => {
-  return  _.isEmpty(value) || (typeof value==='string' && value?.trim?.() === '');
-}
+  return value instanceof Date
```

Comment:
Are we going to be checking each type? each time we have an issue with a new type? :D

Reply:
I agree that a generic solution should be provided
Also reading the code, `isEmptyValue` will always be 'false' is the value is a Date (?)

Reply:
yes, because if there is a Date defined the value is not empty. 
The `._isEmpty()` return true if you give them a date. Do you have any suggestions on how to check it without breaking other cases?